AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -2218,36 +2219,6 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
-		3B9D88CDF51B429C8409E4B6 /* [CP] Copy Pods Resources */ = {
```

Comment:
I could not run the tests due to the Cocoapods build script throwing an error. The error message recommended running `pod install`. The resulting change was the removal of these two run script phases. With them gone, I can now run the tests.

Reply:
Could this be due to differing versions of cocoa pods?

Reply:
I'm not sure what caused it. I have rebased my code changes onto `master` and the issue went away. So I've reverted this project file change.